tests/test_models.py

```diff
@@ -4242,118 +4241,6 @@ def test_node_scale_with_deleted_parent(self):
         link.save()
         assert_equal(link.node_scale(node), -40)
 
-class TestDraftRegistration(OsfTestCase):
```

Comment:
Usually I check this for accidentally deleted code, but if you don't mind, I'm going to just make a blanket assumption that the entire test got correctly moved over in this case. :)